Fix library position and mobile regression.

jasmussen opened the pull request:
This PR fixes two things.

1. When invoking the block library from the plus on the side, it aligns it correctly

2. On mobile breakpoints, the "recently used" blocks were shown on the right even though they should be hidden.

After:

<img width="542" alt="screen shot 2018-08-21 at 09 28 01" src="https://user-images.githubusercontent.com/1204802/44387423-c73f7d80-a525-11e8-9ac0-e85910b6c5bb.png">

<img width="498" alt="screen shot 2018-08-21 at 09 34 55" src="https://user-images.githubusercontent.com/1204802/44387428-c9094100-a525-11e8-900f-2658e10e2674.png">

Before:

<img width="489" alt="screen shot 2018-08-21 at 09 29 02" src="https://user-images.githubusercontent.com/1204802/44387432-cc9cc800-a525-11e8-902c-0d80f83e7791.png">

Commit:
Fix library position and mobile regression.

This PR fixes two things.

1. When invoking the block library from the plus on the side, it aligns it correctly

2. On mobile breakpoints, the "recently used" blocks were shown on the right even though they should be hidden.,

gziolo commented on packages/editor/src/components/inserter-with-shortcuts/style.scss:
```diff
@@ -1,5 +1,5 @@
 .editor-inserter-with-shortcuts {
-	display: flex;
+	//display: flex;
```
Is that intended?

jasmussen replied:
Raaargh no, sorry. One sec fixing.

jasmussen replied:
Sorry about that. Fixed not only this typo, but another one. 

<img width="687" alt="screen shot 2018-08-21 at 10 00 35" src="https://user-images.githubusercontent.com/1204802/44388765-38ccfb00-a529-11e8-94bf-187b2436fcc0.png">

<img width="331" alt="screen shot 2018-08-21 at 10 01 11" src="https://user-images.githubusercontent.com/1204802/44388768-3a96be80-a529-11e8-9759-1c7e2a29cb39.png">

Commit:
Fix typos.

noisysocks commented:
I'm noticing that if I click into the default block appender and then click away from it, the block shortcuts cover the placeholder text:

<img width="510" alt="screen shot 2018-08-22 at 11 10 52" src="https://user-images.githubusercontent.com/612155/44437190-3b4c5500-a5fc-11e8-974e-1f8e90f1d836.png">

noisysocks reviewed (changes requested):
I'm noticing that if I click into the default block appender and then click away from it, the block shortcuts cover the placeholder text:

<img width="510" alt="screen shot 2018-08-22 at 11 10 52" src="https://user-images.githubusercontent.com/612155/44437190-3b4c5500-a5fc-11e8-974e-1f8e90f1d836.png">

Commit:
Address feedback.

Commit:
Address feedback.

jasmussen commented:
Solid review, thank you. I believe I've addressed all points now:

<img width="699" alt="screen shot 2018-08-22 at 09 59 07" src="https://user-images.githubusercontent.com/1204802/44450970-a55ffc80-a5f2-11e8-8d9f-b0cc956a574c.png">

<img width="452" alt="screen shot 2018-08-22 at 09 59 13" src="https://user-images.githubusercontent.com/1204802/44450974-a85aed00-a5f2-11e8-831b-40b853324bc4.png">

<img width="458" alt="screen shot 2018-08-22 at 09 59 19" src="https://user-images.githubusercontent.com/1204802/44450978-a98c1a00-a5f2-11e8-9038-eccbeea1b9db.png">

<img width="730" alt="screen shot 2018-08-22 at 09 59 29" src="https://user-images.githubusercontent.com/1204802/44450981-abee7400-a5f2-11e8-87bc-dde6b240823b.png">

<img width="449" alt="screen shot 2018-08-22 at 10 02 51" src="https://user-images.githubusercontent.com/1204802/44450984-ad1fa100-a5f2-11e8-8dd7-bb5d0925afee.png">

noisysocks reviewed (approved):
It's beautiful 😍

jasmussen commented:
😅
